src/Symfony/Component/HttpFoundation/RedirectResponse.php

```diff
@@ -32,7 +32,7 @@ class RedirectResponse extends Response
      *
      * @see http://tools.ietf.org/html/rfc2616#section-10.3
      */
-    public function __construct(?string $url, int $status = 302, array $headers = [])
+    public function __construct(string $url, int $status = 302, array $headers = [])
```

Comment:
this was only nullable because of a unit test. but this is an error that is not meant to be caught. so it just throws a TypeError instead of `\InvalidArgumentException`